src/PageController.php

```diff
@@ -475,25 +477,20 @@ public function register_page( $options ) {
 	public static function add_nav_item( $options ) {
 		$navigation_enabled = Loader::is_feature_enabled( 'navigation' );
 
-		if ( ! $navigation_enabled ) {
+		if ( ! $navigation_enabled || ! $options['parent'] ) {
```

Comment:
Yes, this would require a change from WC Payments to set the category as `payments`.  If we default items to the `settings` category, then in this case we would end up with multiple `Payment` menu items.